### PR TITLE
Update lesson06 (add exercises)

### PR DESCRIPTION
**Pull request by Dominik1123**
Adds two exercises:

1. Using the sklearn wine dataset. Two bonuses:
   1. Add metric `accuracy` for the fitting (+ potentially other metrics).
   2. Play with `learning_rate` to see its influence.
2. A synthetic dataset following a polar coordinate transformation (see image below). Two bonuses:
   1. Try to fit with `LogisticRegression` to realize that a coordinate transformation is needed. Discussion about implicit feature engineering with neural networks (no transformation needed here).
   2. Play with complexity of neural network architecture and compare to min. architecture required for the wine dataset. The wine dataset contains feature combination which allows for linear decision boundaries, hence complexity can be very low (linear model basically); not so for the polar dataset.

Regarding (2.i), is mention of `LogisticRegression` okay here? Is this discussed before, in previous lessons?

image

### CODE REVIEW

**Comment by Dominik1123** [comment]
> Thanks for the contribution. This is quite a chunk of work. Wow! I am very impressed. I left some comments.

I think I addressed all the comments. Let me know if there's more to do.
